components/calendar/calendar.js

```diff
@@ -133,6 +140,8 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 
 	static get properties() {
 		return {
+			maxValue: { type: String, attribute: 'max-value' },
+			minValue: { type: String, attribute: 'min-value' },
```

Comment:
We should start reflecting new attributes I think, now that we've decided it's the best practice in out Lit guide.